boltons/iterutils.py

```diff
@@ -374,6 +374,30 @@ def chunked_iter(src, size, **kw):
     return
 
 
+def chunked_filter(iterable, predicate, chunk_size):
```

Comment:
Great to see a docstring here. Looking around at other functions, I noticed that most refer to the predicate as `key`. Do you mind updating this argument to be consistent?

Reply:
I see various names. builtins.filter uses `iterable` and `function`. Actually it's [filter(function, iterable)](https://docs.python.org/3/library/functions.html#filter). Since it's a variation of it, how about using these names and order?

Reply:
interestingly [itertools.filterfalse(predicate, iterable)](https://docs.python.org/3/library/itertools.html#itertools.filterfalse)

Reply:
Ah, I think the prime reference for this should be the [`sorted()`](https://docs.python.org/3/library/functions.html#sorted) function, where `key` is used. `min()` and `max()` also use `key`, I think. So, tldr. `key` should be preferred here. :)

Reply:
> Ah, I think the prime reference for this should be the [`sorted()`](https://docs.python.org/3/library/functions.html#sorted) function, where `key` is used.

There's a good reason why sorted and min/max use `key` and `filter` & `filterfalse` don't. When you compare two objects, you either compare the value itself or its part, hence the `key` - attribute, only shorter. When you filter a collection, it can be an arbitrary criterium that's external to the object, so you use `function` (...to be called on an item) or more specifically, `predicate`.
Also, since it's an extension of `builtins.filter`, what's the reason for following naming pattern of any other function?